### PR TITLE
Changed classes api internal to public and fx stream allMarkPrice

### PR DESCRIPTION
**Pull request by VozyakovAV**
1. Changed classes api internal to public
This was necessary so that I could inherit classes.

2. fx stream allMarkPrice
added subcribe to allMarkPrice for CoinFutures

### CODE REVIEW

**Review by HypsyNZ — APPROVED** [review]
Looks good to me

**Review by HypsyNZ — APPROVED** [review]
Looks good to me

**Review by HypsyNZ — APPROVED** [review]
Looks good to me

File: Binance.Net/Binance.Net.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <PropertyGroup>
-    <PackageId>Binance.Net</PackageId>
+    <PackageId>VozyakovAV.Binance.Net</PackageId>
```
Comment (HypsyNZ) [inline Binance.Net/Binance.Net.csproj:8]: Change this

**Review by HypsyNZ — CHANGES_REQUESTED** [review]
Changes are required

File: Binance.Net/Binance.Net.csproj (hunk 1 above)
Comment (VozyakovAV) [reply Binance.Net/Binance.Net.csproj]: Fixed, sorry.

**Review by HypsyNZ — APPROVED** [review]
Thank you

**Comment by JKorf** [comment]
I'd rather not make those classes public. It can be confusing for people to see those types when theyre new to the library, so I'd rather only make classes public which are actually usable

**Pull request closed without merging**